### ISSUE TITLE
Galaxy Install Metadata

### ISSUE THREAD

**Issue by chouseknecht** [ansible/mazer#17]
##### ISSUE TYPE
 - Bug Report

##### GALAXY CLI VERSION
```
Ansible Galaxy CLI, version 0.1.0
Darwin, cyclone.local, 17.5.0, Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64, x86_64
2.7.14 (default, Nov 14 2017, 23:24:24)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.38)] /Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/bin/python2.7
No config file found; using defaults
```

##### CONFIGURATION
NA

##### SUMMARY
We agreed that the installer would write a manifest in each subdirectory created in the content path, where the manifest will track what's currently installed within that path. The name of the file is `.galaxy_install_info`

##### STEPS TO REPRODUCE

```bash
$ ansible-galaxy content-install -t role chouseknecht.content-testing
```
##### EXPECTED RESULTS
Expected to find `.galaxy_install_info` in the `roles` subdir.

##### ACTUAL RESULTS
Missing `.galaxy_install_info`

**Comment by alikins** [comment]
(some disorganized brain dump notes for my reference, nothing pending)



make sure we record some data about package installs

either the .galaxy_install_info as ansible-galaxy does for roles

or something new

needs to allow for remove by role_name/content_name

needs to allow 'list' to work

needs to include version info  for list and upgrade

- install date
- installed version
other?
- installed by user
- installed by what repo/content/name
- which 'fetch' source it came from?
- install_info format version?
for galaxy installs:
  - the api url for the Content and Repo?
- the fetch url  (ie, the github download_url, local fs, etc)


need model for the data that will be persisted

need to figure out where that gets persisted

need to be able to read/deserialize

**Comment by alikins** [comment]
Fixed in ce2015ed4be88bfd0f06bb5a9a8d7d9375400c6d and in the mazer v0.2.0 release.
